hammer/reporting-remediation/reporting/create_cloudtrail_tickets.py

```diff
@@ -46,8 +46,8 @@ def create_tickets_cloud_trail_logging(self):
 
         main_account = Account(region=self.config.aws.region)
         ddb_table = main_account.resource("dynamodb").Table(table_name)
-        jira = JiraReporting(self.config)
-        slack = SlackNotification(self.config)
+        jira = JiraReporting(self.config, module='cloudtrails')
+        slack = SlackNotification(self.config, module='cloudtrails')
```

Comment:
Please inspect L133-145 of this file. I think we can uncomment that lines - we have possibility to disable reporting.

Reply:
Done